Test/SetVariableFromPowerShellDataFile.Tests.ps1

```diff
@@ -0,0 +1,168 @@
+
+#Requires -Version 4
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+
+$context = $null
+$failed = $false
+
+function Init
+{
+    $script:context = $null
+    $script:failed = $false
+}
+
+function GivenDataFile
+{
+    param(
+        $Name,
+        $Content
+    )
+
+    $Content | Set-Content -Path (Join-Path -Path $TestDrive.FullName -ChildPath $Name)
+}
+
+function GivenWhiskeyYml
+{
+    param(
+        $Content
+    )
+
+    $Content | Set-Content -Path (Join-Path -Path $TestDrive.FullName -ChildPath 'whiskey.yml')
+}
+
+function ThenError
+{
+    param(
+        $Regex
+    )
+
+    It ('should write an error') {
+        $Global:Error | Should -Match $Regex
+    }
+}
+
+function ThenTaskFailed
+{
+    It ('should fail') {
+        $failed | Should -Be $true

```

Comment:
Aren't we opting to use the new `-BeTrue` Pester Should operator?

Reply:
Looks like I've go an old version of Pester installed. I'll clear it out and re-install so I pick up this new operator.